source/security/enterprise-authentication.txt

```diff
@@ -0,0 +1,203 @@
+.. _pymongo-enterprise-auth:
+
+Enterprise Authentication Mechanisms
+====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: ldap, encryption, principal, tls
+
+Overview
+--------
+
+MongoDB Enterprise Edition includes the following authentication mechanisms that aren't
+available in MongoDB: 
+
+- :ref:`GSSAPI/Kerberos <pymongo-kerberos>`
+- :ref:`LDAP (Plain) <pymongo-sasl>`
+
+In this guide, you can learn how to authenticate to MongoDB by using these
+authentication mechanisms. To learn about the other authentication mechanisms available
+in MongoDB, see :ref:`<pymongo-auth>`.
+
+.. _pymongo-kerberos:
+
+GSSAPI (Kerberos)
+-----------------
+
+The Generic Security Services API (GSSAPI) provides an interface for Kerberos
+authentication. Select the tab that corresponds to your operating system to learn how
+to use GSSAPI to authenticate.
+
+.. tabs::
```

Comment:
I used tabs here because Unix and Windows are mutually exclusive for most/all users. Let me know if you think it works (I have my doubts).

Reply:
I think it works fine!